packages/admin-panel/src/VizBuilderApp/components/PreviewOptions.js

```diff
@@ -47,6 +49,30 @@ const PaperComponent = styled(MuiPaper)`
   }
 `;
 
+const UploadedFileContainer = styled(FlexSpaceBetween)`
+  width: 100%;
+  padding: 14px;
+  background-color: ${blue[50]};
+  border: 1px solid ${blue[400]};
```

Comment:
Those colours seem close (but not exactly) with the ones specified in the [design](https://www.figma.com/file/SbUSVEzB3RuBljOdBtE0x5/viz-builder-design?node-id=0%3A1):

Design:
![image](https://user-images.githubusercontent.com/20692464/130754522-5661f2bc-11a0-4bd0-afb9-ba717ee23c34.png)

Implementation:
![image](https://user-images.githubusercontent.com/20692464/130754900-112289ce-c89a-4d73-b349-d2789da6aef2.png)

I would like to get @tcaiger 's opinion on that one - is this fine as is, or should I use the exact hex codes from the design?

Reply:
It's very easy to use the exact colous from the design, so I'll do that then!

Reply:
@tcaiger  Forgot to add the margin in the buttons, done now in the most recent commit:

![image](https://user-images.githubusercontent.com/20692464/131276912-397b1091-523a-4f1f-95c9-dea746bd1247.png)